### PR TITLE
Colour similarity

### PR DESCRIPTION
**Pull request by teh**

### CODE REVIEW

File: hogwarts/scss/_article.scss [hunk 1]
```diff
@@ -125,9 +125,41 @@ article {
 
     }
   }
-}
+
+  /* ipython notebook specific styles */
+  // Disgusting hack: We are including the minimized ipython css file
+  // which was compiled from less files as scss to make sure it only
+  // applies to the ipython_notebook parts:
+  @import "_ipython_style";
```
Comment (Keats) [inline hogwarts/scss/_article.scss]: this is going to fuck up quite a bit of design

Comment (Keats) [inline hogwarts/scss/_article.scss]: it already does actually

Comment (Keats) [inline hogwarts/scss/_article.scss]: remove all those lines below + pygments/ipython css and replacing it with

```scss
  // ipython stuff
  .prompt {
      display: none;
  }

  .output_png {
    width: 100%;
    text-align:center
  }

  .output_text {
    pre {
      padding: 0;
    }
  }
```

makes it consistent with the rest of the code samples

Comment (Keats) [inline hogwarts/scss/_article.scss]: only issue being font size matching the text so it might be a bit too big, but can easily be solved

**Comment by Keats** [comment]
Regarding the notebook itself:

-  i.e. instead of a two-dimensional space of (r, g, b) points we will have a one-dimensional space of (r, g, b) points. => detail that?

- and doesn't work e.g. for fashion items with a light-blue background. => and doesn't work for example for fashion items with a light-blue background ? the e.g. feels a bit out of place

File: hogwarts/scss/_article.scss (hunk 1 above)
Comment (teh) [inline hogwarts/scss/_article.scss]: I'm keeping your commit on my branch as discussed.

File: hogwarts/scss/_article.scss [hunk 2]
```diff
@@ -125,9 +125,50 @@ article {
...
+    pre {
+      padding: 0;
+      font-size: 1rem;
+    }
+  }
+
+  .code_cell {
+    pre {
+      font-size: 1rem;
+    }
```
Comment (Keats) [inline hogwarts/scss/_article.scss]: you should put that in the top level article actually, we also want our own code sample to be that size

File: hogwarts/scss/_article.scss [hunk 3]
```diff
@@ -125,9 +125,50 @@ article {
...
+  .output_png {
+    width: 100%;
+    text-align:center;
+    padding: 1rem;
+  }
+
+  .output_text {
+    pre {
+      padding: 0;
+      font-size: 1rem;
```
Comment (Keats) [inline hogwarts/scss/_article.scss]: and if you put it at the top level, you don't need any of these

**Pull request merged**